### PR TITLE
Added valid_until and display_info to PaymentMethod model.

### PR DESCRIPTION
**Pull request by vtemian**

### CODE REVIEW

File: silver/models/payment_methods.py [hunk 1]
```diff
@@ -55,6 +55,9 @@ def as_list(cls):
     verified = models.BooleanField(default=False)
     canceled = models.BooleanField(default=False)
 
+    valid_until = models.DateTimeField(null=True, blank=True)
+    display_info = models.CharField(max_length=256, null=True, blank=True)
```
Comment (bogdanpetrea) [inline silver/models/payment_methods.py]: What is this used for? `TextField` might be better.

Comment (vtemian) [inline silver/models/payment_methods.py]: Just a small info to easily identify a payment method.
For PayPal should be the email and for credit card the last 4 digits.

Comment (vtemian) [inline silver/models/payment_methods.py]: I really should exceed 256 characters.

Comment (bogdanpetrea) [inline silver/models/payment_methods.py]: How about we use https://github.com/silverapp/silver/blob/master/silver/models/payment_methods.py#L140 which can get it's data from the `data` field. In this way we store data in one place and have it in a serializable form.

Comment (vtemian) [inline silver/models/payment_methods.py]: I would pretty much keep it as a total separate think.

Comment (bogdanpetrea) [inline silver/models/payment_methods.py]: Fair enough.

**Pull request merged**